test/e2e/wmco_test.go

```diff
@@ -54,3 +70,27 @@ func setupWMCOResources() error {
 	}
 	return nil
 }
+
+// configureForUpgrade changes the node annotations of all the configured Windows nodes.
+func configureForUpgrade() error {
```

Comment:
How do you guarantee that the operator is not running at this point? I feel this should be done outside of the test suite to simulate the upgrade scenario.